contracts/FiatToken.cdc

```diff
@@ -1,1017 +1,1000 @@
 import Crypto
-import FungibleToken from 0x{{.FungibleToken}}
-import OnChainMultiSig from 0x{{.OnChainMultiSig}}
-
-pub contract FiatToken: FungibleToken {
-
-    // ------- FiatToken Events -------
```

Comment:
All the events are exactly the same. Just changed the access modifiers to `access(all)`